baselines/power_of_choice/README.md

```diff
@@ -0,0 +1,250 @@
+
+### Figure 4a and 4b
+The results in Figure 4a and 4b in the paper refer to FMNIST dataset partitioned using LDA with `alpha=2` and `alpha=0.3`, respectively. 
+
+To reproduce the results in Figure 4a, [generate the clients' dataset](#generate-clients-dataset) with `alpha=2`.
+
+To reproduce the results in Figure 4b, [generate the clients' dataset](#generate-clients-dataset) with `alpha=0.3`.
+
+In both cases, then run:
+
+```bash
+# This will run the experiment using FedAvg strategy
+python -m power_of_choice.main variant="rand"
+```
+
+and
+
+```bash
+# This will produce 3 consecutive runs, using pow-d strategy with d=6,9,15, respectively
+python -m power_of_choice.main --multirun strategy.d=6,9,15
+```
+
+### Figure 6a and 6b
+The results in Figure 6a and 6b in the paper refer to CIFAR10 dataset partitioned using LDA with `alpha=2` and `alpha=0.3`, respectively. 
+
+To reproduce the results in Figure 6a, [generate the clients' dataset](#generate-clients-dataset) with parameters `dataset.dataset="cifar10"` and  `alpha=2`.
+
+Then run:
+
+```bash
+# This will run the experiment using FedAvg strategy
+python -m power_of_choice.main dataset.dataset="cifar10" variant="rand" is_cnn=True
+
+# This will run the experiment using pow-d with d=20 and CK=9
+python -m power_of_choice.main dataset.dataset="cifar10" variant="base" is_cnn=True strategy.d=20 strategy.ck=9
+
+# This will run the experiment using cpow-d with d=20 and CK=9
+python -m power_of_choice.main dataset.dataset="cifar10" variant="cpow" is_cnn=True strategy.d=20 strategy.ck=9
+
+# This will run the experiment using rpow-d with d=60 and CK=9
+python -m power_of_choice.main dataset.dataset="cifar10" variant="rpow" is_cnn=True strategy.d=60 strategy.ck=9
+```
+
+To reproduce the results in Figure 6b, [generate the clients' dataset](#generate-clients-dataset) with parameters `dataset.dataset="cifar10"` and  `alpha=0.3`.
+
+Then run:
+
+```bash
+# This will run the experiment using FedAvg strategy
+python -m power_of_choice.main dataset.dataset="cifar10" variant="rand" is_cnn=True
+
+# This will run the experiment using pow-d with d=12 and CK=9
+python -m power_of_choice.main dataset.dataset="cifar10" variant="base" is_cnn=True strategy.d=12 strategy.ck=9
+
+# This will run the experiment using cpow-d with d=12 and CK=9
+python -m power_of_choice.main dataset.dataset="cifar10" variant="cpow" is_cnn=True strategy.d=12 strategy.ck=9
+
+# This will run the experiment using rpow-d with d=60 and CK=9
+python -m power_of_choice.main dataset.dataset="cifar10" variant="rpow" is_cnn=True strategy.d=60 strategy.ck=9
+```
+
+The above commands would generate results by creating a directory under the following path `outputs/<date>/<dataset_name>_${variant}_d${strategy.d}_CK${strategy.ck}`, containing a `results.pkl` file that you can plot by using the following command:
```

Comment:
update the path if in the end you make the suggested change to `conf/base.yaml`.